python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py

```diff
@@ -347,6 +354,12 @@ def process_static_tables(db_manager: DatabaseManager) -> None:
             drop_bus_records(static_tables)
             insert_data_tables(static_tables, db_manager)
 
+            static_timestamp = int(
+                static_tables["feed_info"].data_table.loc[0, "timestamp"]
+            )
+
+            modify_static_tables(static_timestamp, db_manager)
```

Comment:
Pull out the `timestamp` value from the Schedule being processed and pass to the `modify_static_tables` function for more processing.